specification/dataprotection/resource-manager/Microsoft.DataProtection/DataProtection/BackupVaultResource.tsp

```diff
@@ -223,6 +258,58 @@ interface BackupVaultOperationResults {
   >;
 }
 
+/**
+ * Deleted backup vault operations (available from version 2025-09-01)
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-interface-requires-decorator" "DeletedBackupVaults interface uses manual routing for location-scoped resources"
+@added(Versions.v2025_09_01)
+interface DeletedBackupVaults {
+  /**
+   * Gets a deleted backup vault
+   */
+  @get
+  @route("/subscriptions/{subscriptionId}/providers/Microsoft.DataProtection/locations/{location}/deletedVaults/{deletedVaultName}")
+  get(
+    ...ApiVersionParameter,
+    ...SubscriptionIdParameter,
+    ...LocationParameter,
+
+    /**
+     * Name of the deleted backup vault
+     */
+    @path deletedVaultName: string,
+  ): ArmResponse<DeletedBackupVaultResource> | ErrorResponse;
+
+  /**
+   * Lists deleted backup vaults by location
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation" "Manual routing required for location-scoped resource listing"
+  @list
+  @get
+  @route("/subscriptions/{subscriptionId}/providers/Microsoft.DataProtection/locations/{location}/deletedVaults")
```

Comment:
Same comment.  This is really easy to represent using standard templates, see: https://github.com/Azure/typespec-azure/blob/main/packages/samples/specs/resource-manager/resource-types/location/main.tsp

Using low-level APIs does not encode as much information in the API as templates do, and will lead to deficiencies in generated SDKs.

Reply:
Updated to use templates